knockout-tinymce.coffee

```diff
@@ -1,4 +1,5 @@
 (($, ko) ->
+  cache = ""
```

Comment:
Yeah, I'm totally on board with adding some caching to reduce the number of `getContent` calls. As far as I can tell, there is no way to pollyfill a `WeakMap`, but we could fall back to a simple `Object` key-value store in cases where `WeakMap` doesn't exist